### ISSUE TITLE
Error: Invalid License file for model

### ISSUE THREAD

**Issue by chutson92** [skier233/nsfw_ai_model_server#14]
Keep getting this after every handful of container restarts.  License file is in the same place as it has always been.

```
08:09:29|[INFO]:Loading pipeline: video_pipeline_gentler_river
08:09:30|[INFO]:Loading model gentler_river with batch size 1, 1, 1
Source Error: Padding is incorrect.
08:09:32|[ERROR]:Error loading pipeline video_pipeline_gentler_river: Error: Invalid License file for model ./models/gentler_river.pt.enc
08:09:32|[INFO]:Loading pipeline: image_pipeline_gentler_river
08:09:32|[ERROR]:Error loading pipeline image_pipeline_gentler_river: Error: Model failed to load!
```

**Comment by skier233** [comment]
Are you using the exact command to start the container from the docker installation instructions or a different command? Also what software are you using to manage the containers (ie docker desktop, or any additional software?)

**Comment by chutson92** [comment]
Have this installed via Docker in UnRAID...so unfortunately, I have NOT been using the commands from the instructions.

**Comment by skier233** [comment]
Gotcha. I do have other UnRAID docker users so it should work. What are you using to stop/start the containers. I think I recall an issue with the way that the containers are restarted can mess with the license.

**Comment by chutson92** [comment]
That could be the issue, as I have performed a 'restart' instead of a 'stop' then 'start' command.  

After I get the license replaced, I'll test that out over the next few days to see if the issue comes back up.

**Comment by skier233** [comment]
Yea I would do some testing with that to see if it was the issue. I just found a conversation in my old DM's of a user on UnRAID docker that had a similar issue. We think it might've had been due to clearing out certain container resources and wiping something but I don't think we got the exact root cause. For now what I'd advise is to manually delete that license file in the models folder and pull down a new one and experiment with how you restart from the UnRAID gui as I expect that might be what's causing issues.
